.github/workflows/deploy-archival-snapshots.yml

```
@@ -0,0 +1,41 @@
+name: Run Ansible playbook for archival snapshots
+
+on:
+  schedule:
+    - cron: '0 0 */10 * *'
```

This will wait 10 days between each run. There's a new diff snapshot to upload roughly every day. Let's run the script every day.
```suggestion
    - cron: '0 0 * * *'
```